a8c-files.php

```diff
@@ -832,6 +870,148 @@ function image_resize( $ignore, $id, $size ) {
 		return array( $img_url, $w, $h, $resized );
 	}
 
+	/**
+	 * Filter `cron_schedules` output
+	 *
+	 * Add a custom schedule for a 5 minute interval
+	 *
+	 * @param   array   $schedule
+	 *
+	 * @return  mixed
+	 */
+	public function filter_cron_schedules( $schedule ) {
+		if ( isset( $schedule[ 'vip_five_minutes' ] ) ) {
+			return $schedule;
+		}
+
+		// Not actually five minutes; we want it to run faster though to get through everything.
+		$schedule['vip_five_minutes'] = [
+			'interval' => 180,
+			'display' => __( 'Once every 3 minutes, unlike what the slug says. Originally used to be 5 mins.' ),
+		];
+
+		return $schedule;
+	}
+
+	public function schedule_update_job() {
+		// Only schedule meta update job on production
+		if ( 'production' !== VIP_GO_ENV ) {
```

Comment:
@batmoo with the new `VIP_FILESYSTEM_SCHEDULE_FILESIZE_UPDATE` constant check I added, would this `ENV` be needed anymore? I feel like it's a better to leave that sort of details to deployment configs

Reply:
Yep, I think we can safely drop this.

I guess to for the rollout, we'd do something like this?

```
if ( ! defined( 'VIP_FILESYSTEM_SCHEDULE_FILESIZE_UPDATE' ) ) {
   if ( 'production' !== VIP_GO_ENV ) {
        define( 'VIP_FILESYSTEM_SCHEDULE_FILESIZE_UPDATE', true );
   }
}
```

Reply:
Yeap. This way we can easily deactivate or change batch sizes for individual sites should we need to

Reply:
Awesome, should be good to go after that change.